experimental/compactdecimal/src/compactdecimal.rs

```diff
@@ -307,9 +313,154 @@ impl CompactDecimalFormatter {
     /// ```
     pub fn format_i64(&self, value: i64) -> FormattedCompactDecimal<'_> {
         let unrounded = FixedDecimal::from(value);
-        let log10_type = unrounded.nonzero_magnitude_start();
+        self.format_fixed_decimal(unrounded)
+    }
+
+    /// Formats an integer in compact decimal notation using the default
+    /// precision settings.
```

Comment:
Sure, though the rounding direction discussion would then come back to haunt us :-)

As I wrote, I am also fine with just not having those options, and seeing the sort of thing that YouTube does as an advanced do-your-own-rounding scenario; since we have format_compact_decimal and CompactDecimal, it is much easier for the caller to have their own arbitrarily fancy rounding logic than it was for me with ICU a few years ago.

Reply:
Follow-up filed: https://github.com/unicode-org/icu4x/issues/3929